src/test/resources/org/rust/ide/intentions/fixtures/unwrap_lambda_expr/unwrap_braces.rs

```diff
@@ -0,0 +1,3 @@
+fn main () {
+    |x| { x *<caret> x }
```

Comment:
@mkaput `{ ... }` can only be `RustBlockExprElement` so i can not figure out a way to access lbrace and rbrace element just like [here](https://github.com/JetBrains/kotlin/blob/master/idea/src/org/jetbrains/kotlin/idea/intentions/RemoveBracesIntention.kt#L31-L33) does.